action-sonos.py

```diff
@@ -233,4 +254,5 @@ def playMusic_callback(hermes, intentMessage):
             .subscribe_intent("muteSound4", mute_callback) \
             .subscribe_intent("resumeMusic4", resumeMusic_callback) \
             .subscribe_intent("speakerInterrupt4", speakerInterrupt_callback) \
+            .subscribe_intent("nextSong4", nextSong_callback) \
```

Comment:
We should define the intent names in a file and import from it, I see they are changing constantly with the new versions of the assistants. So at least we have a place where we centralize this. 
I will open an issue if you agree with this.

Reply:
Yes, this is a good idea. Open up an issue. 
We should do the same thing for TTS feedback strings btw.